lib/pages/portal/view/website_view.dart

```diff
@@ -48,9 +52,14 @@ class _WebsiteViewState extends State<WebsiteView> {
   TextEditingController _descriptionRoController;
```

Comment:
I know you didn't write this (it really may be me who wrote it) but I wanted to point out there's really no point in these attributes and methods to be private (have the leading underscore). The State class is private itself so it really doesn't matter.

You can rename these to make them consistent with the (public) attributes you added (select the attribute name and press shift + F6 in Android Studio to rename), and the methods too (_fetchUser, _uploadButton etc.), but you don't have to.